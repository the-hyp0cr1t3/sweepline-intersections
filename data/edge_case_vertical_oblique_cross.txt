2
3 7 7 3
5 7 5 3