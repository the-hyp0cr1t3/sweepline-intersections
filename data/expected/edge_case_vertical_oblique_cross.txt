1
5.0000000000 5.0000000000 1 2

